### ISSUE TITLE
Run bbmustache from command line

### ISSUE THREAD

**Issue by rjcoelho** [soranoba/bbmustache#46]
I need to instantiate templates in files using erlang terms (also in other files). 

I can open a PR with Main func on bbmustache and then we rebar3 escriptalise it (same as we do for other erlang tools like elvis or rebar3). 

Does it make sense ? 
Thanks

**Comment by soranoba** [comment]
ummm....
I couldn't imagine a scene where bbmustache became escript and would be useful, because I think erlang term is very difficult to pass from the command line.
The reason is the handling of single quotes and double quotes is different. 😓

bbmustache works like other mustache libraries, so [mo](https://github.com/tests-always-included/mo) is probably more convenient than bbmustache in this case.

Alternatively, it makes sense to create a thin escript on demand (I do this a lot).
For example, it looks like this. 

```bash
$ cat <<EOF > make_cnf.escript
#!/usr/bin/env escript
%%! -pa _build/default/lib/bbmustache/ebin

main(_) ->
    Ret = bbmustache:render(<<"{{hoge}}">>, [{"hoge", 1}]),
    io:format("~p~n", [Ret]).

EOF
$ chmod +x make_cnf.escript 
$ ./make_cnf.escript 
<<"1">>
```

By the way, benchmark results are created by escript.
script: https://github.com/soranoba/bbmustache/blob/636815b64aecd34e3fd53e1b10d6769c6ee575d6/benchmarks/bench.escript
result: https://github.com/soranoba/bbmustache/tree/master/benchmarks

**Comment by rjcoelho** [comment]
Both template and the erlang terms are in files. My case is I use overlay vargs I want to use to replace other template vars. Eg:
```
#!/usr/bin/env escript
%%! -pa _build/default/lib/bbmustache/ebin

main(_) ->
    {ok, Data} = file:consult("vars.config"),
    {ok, Template}  = file:read_file("file.template)
    Ret = bbmustache:render(Template, Data),
    io:format("~p~n", [Ret]).
```

**Comment by soranoba** [comment]
oh, I see. it sounds good.
I welcome PR.

**Comment by soranoba** [comment]
Solved by #48
